tests/test_issues/test_issue_2224.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from unittest import TestCase
+from linkml_runtime.utils.schemaview import SchemaView
+from jsonasobj2 import JsonObj
+
+from tests.test_issues.environment import env
+
+
+class Issue2224TestCase(TestCase):
+    env = env
+
+    def test_issue_2224_slot_classes(self):
+        sv = SchemaView(env.input_path("linkml_issue_2224.yaml"))
+        cls = sv.induced_class("DJController")
```

Comment:
again this is just a 'if it were up to me thing', but i find organizing tests by issue number to be a bad pattern - it's hard to know what this test is testing for from the test results, and then when evaluating "the tests for schemaview" it's hard to know what tests already exist and what don't, so we end up with multiple tests for the same thing. I would call this something like "test_induced_slot_usage_overwrite" or whatever and then put a description and link to the issue in the docstring so that when the test fails we can be like "oh something is wrong with the way that induced slots are overwriting slot usage," and when we need to test similar things we can ctrl+f around to find where we should put the next test near.

Reply:
Makes complete sense, and good suggestion again! I'll move this test into `tests/test_utils/test_schemaview.py`.